### DISCUSSION TITLE
Error running cookbook example "Comparison with hydrostatics from Meshmagick"

### DISCUSSION THREAD

**Discussion by arredondo-galeana** [capytaine/capytaine#248] [Q&A]
Hi everyone. I am trying to run the cookbook example called: "Comparison with hydrostatics from Meshmagick" but I get the following error on the sphere object.

AttributeError: 'Sphere' object has no attribute 'compute_hydrostatics'

I also got this error trying to run a simpler script, so it seems like the Sphere object does not have such an attribute.
Can anyone help me solving this issue?

Thanks and regards,

Abel

**Comment by mancellin** [comment]
Hydrostatics was added in version 1.4. Could you be using an older one?
```python
import capytaine; print(capytaine.__version__)
```

**Reply by arredondo-galeana** [reply]
I see...yes, it seems like I have an older version (1.3), I will try getting version 1.4. 

Thanks Matthieu!
